src/modules/Login/Login.tsx

```diff
@@ -0,0 +1,100 @@
+import { zodResolver } from '@hookform/resolvers/zod';
+import {
+  Box,
+  Button,
+  Container,
+  Link as MuiLink,
+  TextField,
+  Typography,
+} from '@mui/material';
+import { Link } from '@tanstack/react-router';
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+
+import { useNotifications } from '@/components/Notifications';
+
+import { useLogin } from './api/loginUser';
+import { LoginFormInputsType, loginSchema } from './schema/loginSchema';
+
+export const Login: React.FC = () => {
+  const { t } = useTranslation();
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<LoginFormInputsType>({
+    resolver: zodResolver(loginSchema),
+  });
+
+  const { mutate, isPending } = useLogin({
+    mutationConfig: {
+      onSuccess: (data) => {
+        console.log('Login successful:', data);
+
+        useNotifications.getState().addNotification({
+          type: 'success',
+          titleTranslationKey: 'api.login-form.success',
+        });
```

Comment:
```suggestion
        useNotifications.getState().addNotification({
          type: 'success',
          title: t('api.login-form.success'),
        });
```

you shouldn't pass any translation keys, pass translation returned by `t`

Reply:
Done